src/internal/context/single-tab-stop-navigation-context.tsx

```diff
@@ -97,7 +97,7 @@ export const SingleTabStopNavigationProvider = forwardRef(
       const isUnregisteringFocusedNode = nodeBelongs(focusableElement, document.activeElement);
       if (isUnregisteringFocusedNode) {
         // Wait for unmounted node to get removed from the DOM.
-        requestAnimationFrame(() => onUnregisterActive?.(focusableElement));
+        setTimeout(() => onUnregisterActive?.(focusableElement), 0);
```

Comment:
Why did we change from `requestAnimationFrame` to `setTimeout`?

Reply:
That is due to Wait for unmounted node to get removed from the DOM comment. The requestAnimationFrame fires before the element is removed and the focus is casted on the element which is then removed.

The code is used in button group, table, and tabs. Using the set timeout does not affect the other component but using the animation frame in button group will fail an integration test designed to capture this behavior.

Reply:
Got it, thanks for clarification!